test/test-dockertools.js

```diff
@@ -155,15 +156,18 @@ describe('cloud-enablement:dockertools', function () {
 				});
 				
 				if ( language === "SPRING" ) {
-					it('Dockerfile contains app.jar', function () {
-						assert.fileContent('Dockerfile', '/app.jar');
+					it('Dockerfile contains full jar name and app.jar', function () {
+						assert.fileContent('Dockerfile', `${applicationName}-${javaVersion}.jar /app.jar`);
```

Comment:
Does the `applicationName` need a `.toLowerCase()`?

Reply:
No as the name in the jar file will contain capital letters if the application name has capital letters.